schemas/json/theme.json

```diff
@@ -796,7 +797,28 @@
 					"$ref": "#/definitions/settingsPropertiesComplete"
 				},
 				"core/image": {
-					"$ref": "#/definitions/settingsPropertiesComplete"
+					"type": "object",
+					"allOf": [
+						{
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						{
+							"type": "object",
+							"properties": {
+								"behaviors": {
+									"description": "Allows users to set named behaviors for a block.\nGutenberg plugin required.",
+									"type": "object",
+									"properties": {
+										"lightbox": {
+											"description": "Show a lightbox for the image.\nGutenberg plugin required.",
```

Comment:
> Gutenberg plugin required.

Is this hidden behind an experiment flag? If yes, the schema should not be added at all. If no, this part should be removed as you are editing the plugin version of the schema. We don't do any processing of the schemas for release so it will become outdated when we tag the wp/6.3 version.

Reply:
Makes sense! I've removed the the schema now 👍